### DISCUSSION TITLE
blog/algorithm/2023-10-09-welcome-to-jekyll/

### DISCUSSION THREAD

**Discussion by giscus** [dongdong-119/dongdong-119.github.io#1] [General]
# blog/algorithm/2023-10-09-welcome-to-jekyll/

You’ll find this post in your _posts directory. Go ahead and edit it and re-build the site to see your changes. You can rebuild the site in many different ways, but the most common way is to run jekyll serve, which launches a web server and auto-regenerates your site when a file is updated.

http://127.0.0.1:4000/blog/algorithm/2023-10-09-welcome-to-jekyll/

**Comment by dongdong-119** [comment]
test comment
